CustomDropDown.xcodeproj/xcshareddata/xcschemes/Example.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@FranklinSamboni 
Do we need this file? If not can you add the pattern in .gitignore, may be I missed? Let me know if this is necessary

Reply:
No, It's not necessary, I am going to delete.

Reply:
Thank you! Push the changes, I'll merge.